tests/unit/draggable/core.js

```diff
@@ -338,6 +338,27 @@ QUnit.test( "blur behavior - descendant of handle", function( assert ) {
 	} );
 } );
 
+QUnit.test( "blur behavior - off handle", function( assert ) {
+	var ready = assert.async();
+	assert.expect( 3 );
+
+	var element = $( "#draggable2" ).draggable( { handle: "span" } ),
+		focusElement = $( "<div tabindex='1'></div>" ).appendTo( element );
+
+	testHelper.onFocus( focusElement, function() {
+		assert.strictEqual( document.activeElement, focusElement.get( 0 ), "test element is focused before mousing down on a draggable" );
+
+		testHelper.move( focusElement, 1, 1 );
+
+		assert.strictEqual( document.activeElement, focusElement.get( 0 ), "test element is focused after mousing down on itself" );
+
+		testHelper.move( element, 1, 1 );
+
+		assert.strictEqual( document.activeElement, focusElement.get( 0 ), "test element is focused after mousing down off the handle" );
```

Comment:
Because firing a mousedown event in an element won't force a blur (although a manual mouse interaction will).

The blur in question is manually triggered by the draggable element. This test is meant to make sure the draggable only does its manual blur if the mousedown occurs in the handle; otherwise it shouldn't do any special handling, since the event is otherwise ignored by the control.